mkdocs.yml

```diff
@@ -0,0 +1,11 @@
+site_name: Auto-GPT
+site_description: The coolest thing you have seen this week
+site_url: https://example.com/

```

Comment:
please update to point to our github repo

Reply:
Done